apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -336,17 +338,19 @@ export default class Utilities {
   ): child_process.ChildProcess {
     let shellCommand: string = process.env.comspec || 'cmd';
     let commandFlags: string = '/d /s /c';
+    let useShell: boolean = true;
     if (process.platform !== 'win32') {
       shellCommand = 'sh';
       commandFlags = '-c';
+      useShell = false;
     }
 
     return child_process.spawn(
       shellCommand,
       [commandFlags, command],
       {
         cwd: workingDirectory,
-        shell: true,
+        shell: useShell,
         env: environmentVariables,
         stdio: captureOutput ? ['pipe', 'pipe', 'pipe'] : [0, 1, 2]
       });
```

Comment:
we should refactor to use a base function. I can do that after this if you don't want to do it as part of this PR (since I am the one who introduced the debt)